whatsapp-for-linux.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
```

Comment:
Could you put this file into ui folder? Or we can create a new folder for resources as `resource` maybe?